src/scripts/main.js

```diff
@@ -2,6 +2,48 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const element = document.createElement('div');
+
+  element.style.cssText = `
+  box-sizing: border-box;
+  width: 350px;
+  height: 130px;
+  padding: 0 15px;
+  position: absolute;
+  border-radius: 10px;
```

Comment:
btw, there are ready styles, you could just use proper class names
<img width="742" alt="image" src="https://user-images.githubusercontent.com/102411130/221800216-9feca73a-b0ea-4fb1-826a-dc3c1e06bed1.png">

Reply:
okay, thanks